docs/src/time_integration.md

```diff
@@ -14,3 +14,13 @@ Interesting classes of time integration schemes are
   of stages, e.g. to allow for interpolation (dense output), root-finding for continuous callbacks,
   and error-based time step control. In general, you often should not need to worry about this if you
   use Trixi.
+
+Some common options for `solve` are the following. Further documentation can be found
+in the [SciML docs](https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/).
+- If you use a fixed time step method like `CarpenterKennedy2N54`, you need to pass
+  a time step as `dt=...`. If you use a `CFLCallback`, the value passed as `dt=...`
+  is irrelevant since it will be overwritten by the `CFLCallback`.
+- You should usually set `save_everystep=false`. Otherwise, OrdinaryDiffEq.jl will
```

Comment:
```suggestion
- You should usually set `save_everystep=false`. Otherwise, `OrdinaryDiffEq.jl` will
```

Reply:
I wouldn't like to change that here since we don't use monotype font for Julia packages alsewhere, e.g. the readme. You shouldn't like this change either, Mr. Consistency :wink:

Reply:
Yes, yes, you are right of course!